examples/yaml-to-anydata-with-projection/yaml_to_anydata_with_projection.bal

```diff
@@ -0,0 +1,31 @@
+import ballerina/data.yaml;
+import ballerina/io;
+
+type ServerConfig record {|
+    string host;
+    int port;
+    int[2] remotePorts;
+    DatabaseConfig database;
+|};
+
+type DatabaseConfig record {|
+    string dbName;
+    string username;
+|};
+
+
+final string yamlString =
```

Comment:
Why not use a string template? I would make this a local variable.

Reply:
We can add a comment to say this could be something read from a file.

Reply:
With string template to add new line characters need to add interpolations.

Reply:
No? Actually becomes a bit easier to read.

```ballerina
final string yamlString = string 
`host: "localhost"
port: 8080
remotePorts: [9000, 9001, 9002, 9003]
protocol: "http"
database:
  dbName: "testdb"
  username: "dbuser"
  password: "dbpassword"
`;
```

Reply:
updated.